terraform/deployments/tfc-configuration/variables.tf

```diff
@@ -0,0 +1,289 @@
+#------------------------------------------------------------------------------
+# Organisation
+#------------------------------------------------------------------------------
+variable "tfc_hostname" {
+  type        = string
+  default     = "app.terraform.io"
+  description = "The hostname of the TFC or TFE to use with AWS"
+}
+
+variable "organization" {
+  type        = string
+  description = "Name of Organization to create Workspace in."
+  default     = "govuk"
+}
+
+variable "token" {
+  type        = string
+  description = "Account token"
+}
+
+#------------------------------------------------------------------------------
+# Projects
+#------------------------------------------------------------------------------
+variable "project_names" {
+  description = "List of project names"
+  type        = list(string)
+  default     = ["govuk-infrastructure"]
+}
+
+#------------------------------------------------------------------------------
+# Workspace
+#------------------------------------------------------------------------------
+
+variable "workspace_name" {
+  type        = string
+  description = "Name of Workspace."
+  default     = "Default Workspace"
+}
+
+variable "workspace_desc" {
+  type        = string
+  description = "Description of Workspace."
+  default     = "Created by Terraform Workspacer module."
+}
+
+variable "agent_pool_id" {
```

Comment:
Optional: this and a few others (e.g. file_triggers etc.) look unused — are they? If so, is there value in declaring them preemptively or is it cleaner to leave them out until/unless we find a use for them?

Reply:
There are a few that were not using yet or are the defaults. i'll go through and comment them out for now and then we can add in as needed. Should be easier to see whats been changed that way.